wolox-training/rof-angular
# Implemented profile editing functionalities

## Summary

Implemented profile editing functionalities

## Test Plan

* Log into your account
* Go to profile
* Click on edit fields
* Modify your data
* Refresh webpage to confirm changes

## Trello Card 

https://trello.com/c/1skrk0rU

## Screenshot

![changes](https://cloud.githubusercontent.com/assets/2911971/10701640/01dd8d60-799b-11e5-9527-6cb3922756d7.png)
